exercises/practice/phone-number/test/phone_number_test.cljs

```diff
@@ -3,7 +3,7 @@
             [cljs.test :refer [deftest testing is] :as t :include-macros true]))
 
 (deftest cleans-number
-  (is (= "1234567890" (phone-number/number "(123) 456-7890"))))
+  (is (= "2345678901" (phone-number/number "(234) 567-8901"))))
 
 (deftest cleans-number-with-dots
   (is (= "5558675309" (phone-number/number "555.867.5309"))))
```

Comment:
Interestingly, it seems that these test cases have completely been changed / are not the same as [these canonical tests](https://github.com/exercism/problem-specifications/blob/main/exercises/phone-number/canonical-data.json). This might be the perfect opportunity to update this exercise with those definitions?

Reply:
That would probably be even better. As it is right now a strict implementation of the NANP definition will not test successfully, which is sub-optimal IMO ;-)